Multi tour (#123)
Mapotempo/optimizer-api: CHANGELOG.md

### Added

- Corresponding vehicle_id is returned within each service's skills if problem is partitioned with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)

> - Implementation of vehicle_trips relation : these vehicles' routes are successive [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)

Reviewer: ```suggestion
- Implementation of vehicle_trips relation : these vehicles' routes are successive [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)
```
```suggestion
- Implementation of `vehicle_trips` relation : these vehicles' routes are successive [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)
```

> - Implementation of `vehicle_trips` relation : these vehicles' routes are successive [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)
> - Implementation of lapses within vehicle_trips relations : minimum duration between two consecutive trips [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)

Reviewer 2: ```suggestion
- Implementation of `vehicle_trips` relation: the routes can be successive or with a minimum duration `lapse` in between [#123] (https://github.com/Mapotempo/optimizer-api/pull/123)
```
